### PR TITLE
sortBy multiple columns

### PR DESCRIPTION
**Pull request by NiklasBr**
partially solves #84

### CODE REVIEW

File: src/GraphQL/QueryFieldConfigGenerator/ImageGallery.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ public function getGraphQlFieldConfig($attribute, Data $fieldDefinition, $class
      */
     public function getFieldType(Data $fieldDefinition, $class = null, $container = null)
     {
-        $hotspotType = $this->getGraphQlService()->getTypeDefinition("hotspot");
+        $hotspotType = $this->getGraphQlService()->getTypeDefinition(Hotspotimage::TYPE);
```
Comment (NiklasBr) [inline src/GraphQL/QueryFieldConfigGenerator/ImageGallery.php]: Don't know how this appeared here when the [change has already been merged](https://github.com/pimcore/data-hub/blob/master/src/GraphQL/QueryFieldConfigGenerator/ImageGallery.php)

Comment (weisswurstkanone) [reply src/GraphQL/QueryFieldConfigGenerator/ImageGallery.php]: don't worry ;-)

**Comment by weisswurstkanone** [comment]
is this ready to be reviewed ?

**Comment by NiklasBr** [comment]
Yes it is

**Comment by weisswurstkanone** [comment]
thx!

**Pull request merged**